Podfile

```diff
@@ -288,6 +288,10 @@ target 'ExperimentsTests' do
   experiments_pods
 end
 
+abstract_target 'Tools' do
```

Comment:
This is a fine solution (so no need to change), but I'm wondering if we could use this chance to go straight to SPM.

Reply:
I've been trying the SPM plugin in WordPressKit and WordPressShared. It gives problems when trying to fetch the remote configuration. See https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/348/files#diff-7109ad4ca9c6e705a8da290e7cb2898a679b0f8f1f4712d082dee256230d6a1fR6-R23

Now... one could say "well, we don't have the remote configuration here either," which is a legitimate criticism. Still, I feel like it'd be great to have that option so that the Swift style is homogeneous across projects. I'd love to get more time to maybe see if there's something that can be done at the SwiftLint level. But there's so many other higher leverage things than linting to tackle.

Reply:
Got it, thanks for the details. I haven't tried it with remote configuration myself, so I wasn't really aware of this.

> Still, I feel like it'd be great to have that option so that the Swift style is homogeneous across projects.

I agree!